### PR TITLE
Index fx Validation form

### PR DESCRIPTION
**Pull request by WillyStorm**
For validation, Contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="style.css">
     <script defer src="menu.js"></script>
     <script defer src="modal.js"></script>
+    <script defer src="validation.js"></script>
```
Comment (Kingstalux) [inline index.html:10]: - [ ] Great job so far 👍 but your email validation does not work as I can still submit the form with an email containing a capital letter. Kindly work on this and make sure that when a user inputs an email containing a capital letter, the form is not submitted and an error message should appear close to the submit button 💯

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @WillyStorm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966

## Highlights
✅ Linters are passing 🔥 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kingstalux) [inline index.html:10]: - [ ] You have not made any changes and the validation does not still work. You are selecting the form using an `id` but the form has only a `class`. Kindly verify your code and make sure the validation works and the error message shows as required before submitting your project for review 👍
- Your code:
- validation.js file
Annotation 2022-04-15 101109
- index.html
Annotation 2022-04-15 101206

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @WillyStorm, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966


 ## Highlights
✅ Linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
Hi @WillyStorm , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @WillyStorm

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please make  a new pull request to submit the preserve data project I can't work on the same pull request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**